explorer/src/Pos/Explorer/Web/Server.hs

```diff
@@ -187,6 +193,20 @@ explorerHandlers _sendActions =
 -- API Functions
 ----------------------------------------------------------------
 
+getTotalAda :: ExplorerMode ctx m => m CAda
+getTotalAda = do
+    utxoSum <- EX.getUtxoSum
+    validateUtxoSum utxoSum
+    pure $ CAda $ fromInteger utxoSum / 1e6
+  where
+    validateUtxoSum :: ExplorerMode ctx m => Integer -> m ()
+    validateUtxoSum n
```

Comment:
@Pastafarianist 

@Martoon-00 already said that there is a function which already does this job, can you use this instead https://github.com/input-output-hk/cardano-sl/pull/1639#issuecomment-332477451 ?

Reply:
@Martoon-00 mentioned `unsafeIntegerToCoin`, but I believe it's a bad choice here. It throws different (and more nondescript) error messages and it returns a result which we don't need here. Would you insist on using it?

Reply:
Some safe `integerToCoin` also exists